tests/e2e_tests/inference_endpoints/test_inference_endpoints.py

```diff
@@ -195,21 +196,50 @@ def test_prediction_with_certificates_and_wrong_subject_name():
     assert prediction_response == "Failed"
 
 
-@pytest.mark.skip
 def test_update_subject_name():
+    start_time = time.time()
     params = {'subjectName': 'client'}
     endpoint_response = update_endpoint(params)
     assert "patched" in endpoint_response.text
     assert endpoint_response.status_code == 200
-    time.sleep(10)
-    running, pod_name = wait_endpoint_setup()
-    endpoint_info.pod_name = pod_name
-    assert running is True
+    updated = wait_ingress_setup(ING_NAMESPACE, TENANT_NAME, ENDPOINT_NAME, start_time)
+    assert updated is True
+    subject_name = get_ingress_subject_name(ENDPOINT_NAME, TENANT_NAME)
+    assert subject_name == 'CN=client'
     return endpoint_response
 
 
+def wait_ingress_setup(ingress_namespace, endpoint_namespace, endpoint_name, start_time):
+    ingress_pod_name = get_ingress_pod_name(ingress_namespace)
```

Comment:
There's no waiting between update call and calling this method. Isn't it possible that Kubernetes won't create updated ingress pod before `get_ingress_pod_name` call? I assume that if it won't, we will get Exception in 
`logs = get_logs_of_pod(ingress_namespace, ingress_pod_name, since_seconds=timer)` below.  Then in every iteration we get into except block, log the same error and wait 10 sec in every iteration. Is that right or am I missing something?

Reply:
Kubernetes is not creating new pod for ingress, so there's no need to handle this situation